spec/core_functions/meta/mixin_exists.hrx

```diff
@@ -0,0 +1,384 @@
+<===> same_module/global/input.scss
+@mixin global-mixin() {}
+
+a {b: mixin-exists(global-mixin)}
+
+<===> same_module/global/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/local/input.scss
+a {
+  @mixin local-mixin() {}
+  b: mixin-exists(local-mixin);
+}
+
+<===> same_module/local/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/through_import/input.scss
+@import "other";
+a {b: mixin-exists(global-mixin)}
+
+<===> same_module/through_import/other.scss
+@mixin global-mixin() {}
+
+<===> same_module/through_import/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/non_existent/input.scss
+a {
+  b: mixin-exists(non-existent);
+}
+
+<===> same_module/non_existent/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> different_module/options.yml
+---
+:todo:
+- sass/libsass#2807
+- sass/dart-sass#554
+
+<===>
+================================================================================
+<===> different_module/undefined/input.scss
+@use "sass:color";
+a {b: mixin-exists("c", "color")}
+
+<===> different_module/undefined/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> different_module/defined/input.scss
+@use "other";
+a {b: mixin-exists("c", "color")}
```

Comment:
Should this be `mixin-exists("c", "other")`?

Reply:
It sure should!